test/reduce.cpp

```diff
@@ -508,7 +509,7 @@ TYPED_TEST(Reduce, Test_Any_Global)
     }
 }
 
-TEST(MinMax, NaN)
+TEST(MinMax, MinMaxNaN)
```

Comment:
Nit: this doesn't really add information to the test name. I am ok with it going in.